packages/perseus/src/util/graphie.ts

```diff
@@ -581,17 +587,135 @@ class Graphie {
         $.extend(this.currentStyle, processed);
     }
 
-    label(point: any, text: any, direction: any, latex?: any) {}
+    grid(
+        xr: Interval,
+        yr: Interval,
+        styleAttributes?: Record<string, any>,
+    ): RaphaelSet {
+        throw new Error("grid called on uninitialized Graphie");
```

Comment:
*thought*: I am pretty sure we'll be able to get rid of these stubs once we get the `drawingTools` array fixed up, but I noticed we are handling these stubs in two different ways.  This stub throws whereas the `circle()` one just has a comment that its overwritten. Maybe always throw in the stub?